microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -0,0 +1,64 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="olm-with-microshift"]
+= Using Operator Lifecycle Manager with {microshift-short}
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-operators-olm
+
+toc::[]
+
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You can add your own catalogs for the Operators you want to use with your applications.
+* You must use the CLI to conduct OLM activities with {microshift-short}. The console and OperatorHub GUIs are not available.
+* You must configure the system to use a mirror by updating your CRI-O configuration directly because `ImageContentSourcePolicy` is not supported in {microshift-short}. See link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{product-version}/html/installing/microshift-deploy-with-mirror-registry#microshift-configuring-hosts-for-mirror_microshift-deployment-mirror[Configuring hosts for mirror registry access] for details.
```

Comment:
Configuring the system to use a mirror is not a *must*. That depends on the catalogsource. If the catalog source (and the content) points to private registries, yes, then it is a must. But if my catalogsource points to registries that are already reachable, then it is fine. I suggest to remove it from here in the very prominent beginning and add it to the catalog discussion, e.g. as a info box reminder ("If your catalog source or the content theirin is in a custom registry that requires special configuration, keep in by to updating your CRI-O configuration .......")